src/app/lib/models/webtorrent_stream_info.js

```diff
@@ -0,0 +1,97 @@
+(function (App) {
+    'use strict';
+
+    var WebTorrentStreamInfo = Backbone.Model.extend({
+        initialize: function () {
+
+            this.on('change:size', function () {
+                this.set('sizeFormatted', Common.fileSize(this.get('size')));
+            }.bind(this));
+
+            this.set('size', 0);
+        },
+
+        selectFile: function () {
+
+            var wtTorrent = this.get('wtTorrent'),
```

Comment:
Its Torrent object from WebTorrent, which i used to get info about upload/download.